Boa.Constrictor/Screenplay/Conditions/Collection/IsACollectionOfType.cs

```diff
@@ -0,0 +1,95 @@
+namespace Boa.Constrictor.Screenplay
+{
+    /// <summary>
+    /// Methods to check if the collection items satifies the given conditions.

```

Comment:
Let's be a little more precise: "Builder methods for Conditions that check if ..."